daemon/pool.go

```diff
@@ -245,7 +245,12 @@ func (dp *DriverPool) spawnOne() {
 		case <-stop:
 			return // cancel
 		case dp.spawnErr <- err:
-		case <-ticker.C:
+		case _, ok := <-ticker.C:
+			if !ok {
+				dp.Logger.Errorf("driver keeps failing, closing the pool; error: %v", err)
+				go dp.Stop()
```

Comment:
Why is this in a separate goroutine? If we're closing down anyway I don't think we care about efficiency, and presumably we want to actually ensure this happens before returning?

Reply:
`Stop` will [wait](https://github.com/bblfsh/bblfshd/blob/master/daemon/pool.go#L725) for all goroutines to return, including the spawn goroutine. Because of this, it should be done asynchronously.

Will add a comment explaining this.

Reply:
Done.